R/dragen_fusions.R

```diff
@@ -0,0 +1,28 @@
+#' Read DRAGEN Fusions
+#'
+#' Reads the `fusion_candidates.final` file output by DRAGEN.
+#' @param x Path to the `fusion_candidates.final` file.
+#' @return A tibble with the contents of the input TSV file, or NULL if input is NULL.
+#' @examples
+#' x <- system.file("rawdata/test_data/dragen",
+#'   "test_sample_WTS.fusion_candidates.final",
+#'   package = "RNAsum"
+#' )
+#' (a <- dragen_fusions_read(x))
+#' @testexamples
+#' expect_equal(colnames(a)[ncol(a)], "ReadNames")
+#' expect_equal(colnames(a)[1], "gene1")
+#' expect_null(dragen_fusions_read(NULL))
+#' @export
+dragen_fusions_read <- function(x = NULL) {
+  if (is.null(x)) {
+    return(NULL)
+  }
```

Comment:
Is this the test you had mentioned in the PR description?

Reply:
These NULL checks essentially handle the case when a param is not provided.
Here's an example:

```r
param <- list(
  a = "one",
  b = "two"
)

param$a # "one"
param$dragen_fusion # NULL
```

So then you have `df <- dragen_fusions_read(params$dragen_fusion)` returning NULL, and then you have your chunk runner being:

```
runDragenFusionChunk <- !is.null(df) # FALSE
```

That's the general idea ;-)